ide/parsing.indexing/apichanges.xml

```diff
@@ -87,6 +87,20 @@ is the proper place.
 <!-- ACTUAL CHANGES BEGIN HERE: -->
 
   <changes>
+      <change id="ErrorsCache.getErrors">
+          <api name="IndexingAPI"/>
+          <summary>Added method to ErrorsCache to return all errors or warnings for the given file</summary>
+          <version major="9" minor="37"/>
```

Comment:
I can't find spec version bump in `project.properties`

Reply:
Oops, thanks for pointing this out.